topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,467 @@
+---
+layout: tutorial_hands_on
+level: Intermediate
+title: Train and Test a Deep learning image classifier with Galaxy-Ludwig
+questions:
+- How do I use the Ludwig tool to create and run a deep learning image classifier? 
+- What files do I need to run Ludwig? 
+- How do I build and evaluate the image classification model? 
+objectives:
+- Execute a Ludwig experiment. 
+- Evaluate the image classfier model. 
+- Compare different configurations to improve model performance. 
+time_estimation: 40M
+key_points:
+- Use Galaxy-Ludwig Experiment to solve an image classification problem
+- Using CNN on MNIST dataset. 
+contributors:
+- paulocilasjr 
+- qchiujunhao 
+- jgoecks 
+tags:
+- MNIST
+- Deep learning
+- Ludwig
+---
+
+> <comment-title>Galaxy-Ludwig Tool</comment-title>
+>
+> The Ludwig tool described in this tutorial is only available at: 
+> ```https://cancer.usegalaxy.org/```
```

Comment:
Any reason for this? I don't see the tool in the tool shed either, are there plans to add this? Adding the tool to IUC would be ideal, so that other Galaxies can also support this tutorial.

Reply:
secondly, you probably want that url to be a proper hyperlink, not a comment

Reply:
Tomorrow, I'm going to have a more precise answer regarding the point you raised.

Reply:
I added the toll in the tools shed (https://toolshed.g2.bx.psu.edu/view/paulo_lyra_jr/ludwig_applications/3e565bbe8b71)

Not sure we can get this tool (+ original Galaxy-ML tools) into the IUC yet.